packages/components/src/utils/eventUtils.ts

```diff
@@ -42,6 +42,21 @@ export const isEventCancelled = (event: EventFields): boolean => {
   return event.eventStatus === EVENT_STATUS.EVENT_CANCELLED;
 };
 
+/**
+ * Check is event owned by City of Helsinki
+ * @param event
+ * @return {boolean}
+ */
+export const isEventHelsinkiCityOwned = (event: EventFields): boolean => {
+  const publisherPrefix = event?.publisher?.split(':')[0] ?? '';
+  // Prefixes of organization IDs under City of Helsinki root organization 'ahjo:00001',
+  // see https://api.hel.fi/linkedevents/v1/organization/ahjo:00001/
+  //
+  // Kultus subtree of the organization tree starts at 'kultus:0' and is excluded,
+  // see https://api.hel.fi/linkedevents/v1/organization/kultus:0/
+  return ['ahjo', 'helsinki'].includes(publisherPrefix);
```

Comment:
These should be in some constant.

Reply:
Taken into account in ea2b90da2f5c23a2b3a06d4586b31e6e964c9caa